### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by PaulHowley**
Initial commit with computerPlay and playRound functions

### CODE REVIEW

File: index1.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      let playerSelection = prompt("Please input rock, paper or scissors");
+      playerSelection = playerSelection.toLowerCase();
+
+      let computerSelection = computerPlay();
+
+      function playRound(playerSelection, computerSelection) {
+        let result;
+        if (playerSelection === computerSelection) {
+            result = "Draw!";
+          }
```
Comment (krisquigley) [inline index1.html]: Have these on the same line. i.e. `} else if ((playerSelection === "rock" && computerSelection === "paper") ...`

File: index1.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="eng">
+  <head>
+    <meta charset="utf-8">
+    <title>Project: Rock, Paper, Scissors.</title>
+  </head>
+  <body>
+    <script>
+
+      function computerPlay() {
+        let randomDecimal = Math.random();
```
Comment (krisquigley) [inline index1.html]: Use `let` if you plan to reassign a variable, if the value should never change after the initial value, then use `const`

File: index1.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        let randomNumToThree = randomDecimal * 3;
+        let roundedNum = Math.floor(randomNumToThree);
+
+        const weapons = ["rock", "paper", "scissors"];
```
Comment (krisquigley) [inline index1.html]: In JS, we use `'` for any string declarations, rather than `"`

Comment (PaulHowley) [reply index1.html]: Amendments now made Kris. Thanks.

**Review by krisquigley — APPROVED** [review]

**Pull request merged**